### ISSUE TITLE
Feature Request : Smooth Scrolling

### ISSUE THREAD

**Issue by tofl** [geany/geany#1034]
Hello,

Would it be possible to have smooth scrolling enabled in Geany ? Currently, scrolling is done line by line, and it's very painful especially when it's a big file, because the eyes are not fast enough to "adapt".

**Comment by elextr** [comment]
I don't believe that the underlying [Scintilla library](http://www.scintilla.org/ScintillaDoc.html) that Geany uses supports anything other than scrolling by line.

**Comment by tofl** [comment]
Would it be possible to add this through a plugin ? There has to be a way to add this functionality, isn't there ?

**Comment by eht16** [comment]
It's rather not possible or if at all very hard to implement this as a plugin. It rather has to be supported by Scintilla. But its author won't implement it:
https://sourceforge.net/p/scintilla/feature-requests/688/
But as Neil, the author of Scintilla stated, if someone will contribute a patch, it could get in.

See also http://lists.geany.org/pipermail/users/2012-October/008362.html.
